mapiproxy/servers/default/nspi/dcesrv_exchange_nsp.c

```diff
@@ -235,29 +235,74 @@ static void dcesrv_NspiUnbind(struct dcesrv_call_state *dce_call,
 
 	DCESRV_NSP_RETURN(r, 1, NULL);
 }
+
+static enum MAPISTATUS position_in_table(TALLOC_CTX *mem_ctx, struct STAT *pStat, 
+					 struct emsabp_context *emsabp_ctx, struct PropertyTagArray_r *mids,
+					 uint32_t *out_row, uint32_t *out_last_row)
+{
+	enum MAPISTATUS			retval;
```

Comment:
Personally I don't see the point of this variable, you are not using it as a `MAPISTATUS` ret value, a `bool found = false` would make much more sense